src/DebugEngineHost/HostNatvisProject.cs

```diff
@@ -81,6 +167,22 @@ public static void FindNatvisInSolutionImpl(List<string> paths)
                     return; // failed to find a solution
                 }
 
+                object openFolderMode;
+                solution.GetProperty((int)__VSPROPID7.VSPROPID_IsInOpenFolderMode, out openFolderMode);
+                bool isOpenFolderActive = (bool)openFolderMode;
+
+                if (isOpenFolderActive)
+                {
+                    bool isIndexComplete;
+                    Array filenames;
+                    GetOpenFolderSourceLocations(".natvis", out isIndexComplete, out filenames);
+                    foreach (var f in filenames)
+                    {
+                        paths.Add((string)f);
```

Comment:
If you use IEnumerable<string> for filenames, then you can also use paths.AddRange(filenames) here.